plugins/com.google.cloud.tools.eclipse.appengine.ui.test/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroupTest.java

```diff
@@ -66,6 +69,29 @@ public void run() {
   }
 
   @Test
+  public void testButtonOrder() {
+    syncExec(new Runnable() {
+
+      @Override
+      public void run() {
+        Control groupAsControl = shell.getChildren()[0];
+        assertThat(groupAsControl, instanceOf(Group.class));
+        Control[] buttonsAsControls = ((Group)groupAsControl).getChildren();
+        String[] expectedLibraryOrder = new String[]{ "appengine-api", "appengine-endpoints", "objectify" };
+        for (int i = 0; i < buttonsAsControls.length; i++) {
+          Control control = buttonsAsControls[i];
+          assertThat(control, instanceOf(Button.class));
+          Button button = (Button)control;
+          assertNotNull(button.getData());
+          assertThat(button.getData(), instanceOf(Library.class));
```

Comment:
you're right, it also throws AssertionError (I assumed there would be an NPE which was false). I remove the nullcheck.